src/main/scala/org/broadinstitute/dsde/firecloud/utils/TSVFormatter.scala

```diff
@@ -11,102 +10,98 @@ import spray.json.JsValue
 
 object TSVFormatter {
 
-  def makeMembershipTsvString(entities: List[EntityWithType], entityType: String, collectionMemberType: String): String = {
+  def makeMembershipTsvString(entities: Seq[RawlsEntity], entityType: String, collectionMemberType: String): String = {
     val headers: immutable.IndexedSeq[String] = immutable.IndexedSeq("membership:" + entityType + "_id", entityType.replace("_set", "_id"))
-    val rows: List[IndexedSeq[String]] = entities.filter { _.entityType == entityType }.flatMap {
+    val rows: Seq[IndexedSeq[String]] = entities.filter { _.entityType == entityType }.flatMap {
       entity =>
-        entity.attributes.getOrElse(Map.empty).filter {
+        entity.attributes.filter {
           // To make the membership file, we need the array of elements that correspond to the set type.
           // All other top-level properties are not necessary and are only used for the data load file.
-          attribute => attribute._1.equals(collectionMemberType) }.flatMap {
-          m =>
-            Try(m._2.convertTo[List[JsValue]]) match {
```

Comment:
@breilly2 GAWB-1305 was fixed here. The old code attempted to treat the attribute value as a list and if it couldn't just return IndexedSeq.empty (which is why the file was empty). The new way doesn't touch json so the problem is no longer possible.